client/src/main/java/org/apache/celeborn/client/read/DfsPartitionReader.java

```diff
@@ -215,10 +215,8 @@ public ByteBuf next() throws IOException {
         logger.debug("poll result with result size: {}", results.size());
       }
     } catch (InterruptedException e) {
-      Thread.currentThread().interrupt();
-      IOException ioe = new IOException(e);
-      exception.set(ioe);
-      throw ioe;
+      logger.error("PartitionReader thread interrupted while fetching data.");
```

Comment:
When ```next()``` throws InterruptedException, the inputstream should not retry partition locations. Maybe we can fix this in later PR.